src/utils/is-ref-object.ts

```diff
@@ -1,5 +1,5 @@
-import { RefObject } from "react"
+import { MutableRefObject } from "react"
 
-export const isRefObject = (ref: any): ref is RefObject<Element> => {
+export function isRefObject<E = any>(ref: any): ref is MutableRefObject<E> {
     return typeof ref === "object" && ref.hasOwnProperty("current")
```

Comment:
According to Onne this is best called like `Object.prototype.hasOwnProperty.call(object, "value")`

https://framer-team.slack.com/archives/C01B14R6E22/p1613486064116100?thread_ts=1613472407.103500&cid=C01B14R6E22